release/models/local-routing/openconfig-local-routing.yang

```diff
@@ -225,19 +232,49 @@ module openconfig-local-routing {
         only support a numeric value for this string. ";
     }
 
+    leaf next-network-instance {
+      description
+        "The network instance in which context packet destination address
+        lookup shall be performed.
+        This leaf is mutualy exclusive with 'next-hop', 'nh-network-instance',
+        'recurse' leaves";
+      type leafref {
```

Comment:
```suggestion
        "Instead of finding the next-hop for a destination address in the current
        network-instance, look up the destination address in the routing table
        of the specified network-instance.
        For example, let 'nh-network-instance' = 'DEFAULT'.  If a packet arrives in
        the 'BLUE' network-instance and the destination address matches this
        route, the destination address will be looked up again in the 'DEFAULT'
        network-instance to find the next-hop.  
        This leaf is mutually exclusive with 'next-hop', 'nh-network-instance',
        'recurse' leaves";
      type leafref {
```

Reply:
Adding an example helps clarify the description I think.

Reply:
What if the next-hop in the next-network-instance is an IP address?  Shouldn't recurse be allowed?